### ISSUE TITLE
Update tnoodle-lib to 0.19.2

### ISSUE THREAD

**Issue by Digipr01** [jhale1805/tnoodle-cli#2]
Can this be updated to work with the newest verskon of tnoodle? (0.19.2)

**Comment by thehale** [comment]
Does updating the dependency and rebuilding accomplish what you're hoping?

https://github.com/SpeedcuberOSS/tnoodle-cli/blob/main/build.gradle#L26

**Comment by Digipr01** [comment]
yes, if that is possible then that would be absolutely awesome. This was like almost the only way I've managed to get Tnoodle working for a discord bot. Tnoodle-lib sends me to some web page and idk how to go around that

**Comment by thehale** [comment]
[Version 1.1.0](https://github.com/SpeedcuberOSS/tnoodle-cli/releases/tag/v1.1.0) contains the updated version of TNoodle. Let me know how it works for you!
